$NetBSD: patch-al,v 1.2 2004/09/21 17:38:53 xtraeme Exp $

Workaround to make this build on Linux, I think the error is related
to buildlink.

--- programs/Xserver/hw/dmx/dmxinit.c.orig	2004-09-16 15:20:43.000000000 +0200
+++ programs/Xserver/hw/dmx/dmxinit.c	2004-09-16 15:21:21.000000000 +0200
@@ -573,7 +573,7 @@
 #endif
 
     if (dmxGeneration != serverGeneration) {
-	int vendrel = VENDOR_RELEASE;
+	int vendrel = NULL;
         int major, minor, year, month, day;
         
         dmxGeneration = serverGeneration;
@@ -595,7 +595,7 @@
         dmxLog(dmxInfo, "DMX version:        %d.%d.%02d%02d%02d (%s)\n",
                major, minor, year, month, day, VENDOR_STRING);
 
-        SetVendorRelease(VENDOR_RELEASE);
+        SetVendorRelease(NULL);
         SetVendorString(VENDOR_STRING);
 
         if (dmxGeneration == 1) {
